src/main/java/core/basesyntax/dao/impl/SmileDaoImpl.java

```diff
@@ -12,16 +15,42 @@ public SmileDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Smile create(Smile entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't create smile " + entity, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public Smile get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.get(Smile.class, id);
+        } catch (Exception e) {
+            throw new RuntimeException("Smile with id= " + id + "doesn't exits", e);
```

Comment:
in all your classes you have different exception messages for same operations

it looks not consistent - let's choose one style and apply it to all messages